package.json

```diff
@@ -44,8 +44,8 @@
     "test:compatibility:watch": "karma start --compatibility all --auto-watch=true --single-run=false",
     "test:update-snapshots": "karma start --update-snapshots",
     "test:prune-snapshots": "karma start --prune-snapshots",
-    "test:bs": "karma start karma.bs.config.js --coverage --compatibility all",
-    "test:bs:compatibility": "karma start karma.bs.config.js --coverage --compatibility all",
+    "test:bs": "karma start karma.bs.config.js --coverage",
+    "test:legacy:bs":"karma start karma.bs.config.js --legacy --browsers=bs_win10_ie_11",
```

Comment:
Does this work for you?
When I try to run this command I get a lot of:
```
IE 11.0.0 (Windows 10.0.0) LOG: 'Error loading test file: /packages/overlays/test/OverlayController.test.js'
IE 11.0.0 (Windows 10.0.0) LOG: 'Error loading test file: /packages/radio-group/test/lion-radio-group.test.js'
...
```

Also just before the errors above, I get the following warning:

`testing-karma --legacy flag has been renamed to --compatibility`
https://github.com/open-wc/open-wc/blob/master/packages/testing-karma/src/create-default-config.js#L4

@daKmoR do you know the source of the above?

Reply:
No, i have the same problem as you.

Reply:
We have an issue regarding this #108.

Reply:
It is as the logging says, `--legacy` was renamed to `--compatibility` in `@open-wc/testing-karma`. This change to the package.json shouldn't be there, probably your dependencies are not up to date.

Reply:
Thank you for info, i will update my branch with master and retry.